Package.swift

```diff
@@ -25,6 +25,7 @@ let package = Package(
     .package(url: "https://github.com/OpenCombine/OpenCombine.git", from: "0.10.0"),
     .package(url: "https://github.com/vapor/vapor.git", from: "4.29.3"),
     .package(url: "https://github.com/apple/swift-crypto.git", from: "1.1.0"),
+    .package(url: "https://github.com/JohnSundell/Splash", from: "0.14.0"),
```

Comment:
I'm not sure this will fix build failures with Xcode 11.7, but still would be great to make it consistent.

```suggestion
    .package(url: "https://github.com/JohnSundell/Splash.git", from: "0.14.0"),
```